src/Facebook/Facebook.php

```diff
@@ -327,7 +350,7 @@ public function getRedirectLoginHelper()
    */
   public function getJavaScriptHelper()
   {
-    return new FacebookJavaScriptHelper($this->app);
+    return new FacebookJavaScriptHelper($this->app, $this->getOAuth2Client());
```

Comment:
OK for me, I'll don't say that it's perfect, but I don't have an other solution, so go for it! :)